app/views/internal/page_redirects/edit.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= link_to "Back to All Page Redirects", internal_page_redirects_path, class: "btn btn-primary float-right" %>
+
+<h2>Page Redirect - ID:  <%= @page_redirect.id %></h2>
+<% badge_type =
+     if @page_redirect.version <= 3
+       "success"
+     elsif @page_redirect.version <= 5
+       "warning"
+     else
+       "danger"
+     end %>
```

Comment:
? What does this do?

Reply:
```erb
<% badge_type =
   if @page_redirect.version <= 3
     "success"
   elsif @page_redirect.version <= 5
     "warning"
   else
     "danger"
   end %>
```
```erb
<h4 class="d-inline-block mb-5">
  Version:
  <span class="align-top badge badge-<%= badge_type %>"><%= @page_redirect.version %></span>
</h4>
```

It sets the `badge_type` so the version number displays with the appropriate bootstrap class. For example, a badge with a class of `badge-success` displays as the green/success badge.

<img width="1180" alt="edit_view" src="https://user-images.githubusercontent.com/15987080/83277849-a9925400-a1a0-11ea-84d5-950c692242b7.png">